Gems/Multiplayer/Code/Source/Editor/MultiplayerEditorSystemComponent.cpp

```diff
@@ -328,9 +325,8 @@ namespace Multiplayer
             return;
         }
         
-        AZStd::string sending_leveldata_message = "Editor is sending the editor-server the level data packet.";
-        AZ::Interface<IMultiplayerConnectionViewportMessage>::Get()->DisplayCenterViewportMessage(("(3/3) " + sending_leveldata_message).c_str());
-        AZ_Printf("MultiplayerEditor", sending_leveldata_message.c_str())
+        MultiplayerEditorServerNotificationBus::Broadcast(&MultiplayerEditorServerNotificationBus::Events::OnEditorSendingLevelData);
+        AZ_Printf("MultiplayerEditor", "Editor is sending the editor-server the level data packet.")
```

Comment:
NP: Favor AZ_TracePrintf for the utility around disabling its output

Reply:
cool cool fixed!